src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenUsingVariableTests.cs

```diff
@@ -0,0 +1,348 @@
+﻿using Roslyn.Test.Utilities;
+using Xunit;
+
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests.CodeGen
```

Comment:
I added some test ideas to the test plan (thanks Andy for creating) and will be adding more: https://github.com/dotnet/roslyn/issues/28588
Feel free to edit and document if a test covers a certain scenario, then we'll check off the bullet as "validated".

Nit: double empty line above.